tests/test_q.py

```diff
@@ -119,10 +119,16 @@ class TestQCall(TestCase):
     def setUp(self) -> None:
         super().setUp()
         self.int_fields_context = ResolveContext(
-            model=IntFields, table=IntFields._meta.basequery, annotations={}, custom_filters={}
+            model=IntFields,
+            table=IntFields._meta.basequery,  # type:ignore[arg-type]
+            annotations={},
+            custom_filters={},
         )
         self.char_fields_context = ResolveContext(
-            model=CharFields, table=CharFields._meta.basequery, annotations={}, custom_filters={}
+            model=CharFields,
+            table=CharFields._meta.basequery,  # type:ignore[arg-type]
```

Comment:
Why are there so many ignores added in this PR?

Reply:
I'm curious why these weren't an issue before and they are now. Have you looked into that?

Reply:
Because I add the `py.typed` file to pypika-tortoise